tests/probes/selinuxsecuritycontext/test_probes_selinuxsecuritycontext_offline_mode.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
```

Comment:
Is there any differences compared to the `test_probes_selinuxsecuritycontext.xml`? If not, I would reuse it for offline test and remove this file.

Reply:
Fixed